python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -189,6 +189,18 @@ class DetectAndMeasureConfig(pipeBase.PipelineTaskConfig,
                  "base_PixelFlags_flag_saturatedCenterAll",
                  ),
     )
+    clearMaskPlanes = lsst.pex.config.ListField(
+        dtype=str,
+        doc="Mask planes to clear before running detection.",
+        default=("DETECTED", "DETECTED_NEGATIVE", "NOT_DEBLENDED", "STREAK"),
```

Comment:
Just to check: these fields are set because they were propagated from the science image?

Reply:
Correct